modules/database/export.bal

```diff
@@ -318,24 +318,12 @@ public isolated transactional function getExportDataList(int experimentId, Exper
     ExperimentDataExport[] experimentDataList;
     if config.restriction == "DATA" {
         // only need data files
-        boolean allVersions = config.allDataVersions < 0 ? false : true;
+        boolean allVersions = config.allDataVersions != 0;
```

Comment:
Is this change consistent with the API and the caller? (have you already adapted the UI to this change)
Also, what was the reason for not using a boolean here?